EasyPost.nuspec

```diff
@@ -3,7 +3,7 @@
     <metadata>
         <id>EasyPost-Official</id>
         <title>EasyPost (Official)</title>
-        <version>5.6.0</version>
+        <version>5.6.1</version>
```

Comment:
we might want to do 5.7.0 since there is a new feature (adding missing parameters)

Reply:
But it is a "bug" that they were missing in the first place.

Reply:
A bug is something that doesn't work. Users are able to continue sending requests without the Parameter set as is since they are helper utilities to make that piece easier. Adding (even missing) options to that is a new feature and as such constitutes a minor version bump.

It's imperative we follow the structure of semver closely.